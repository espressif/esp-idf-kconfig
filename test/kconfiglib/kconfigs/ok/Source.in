# test [o|r|or]source entries
mainmenu "Source mainmenu"
    source "$TEST_FILE_PREFIX/sourced"
    rsource "kconfigs_for_sourcing/rsourced"
    osource "$TEST_FILE_PREFIX/osourced"
    osource "test/kconfiglib/kconfigs/ok/kconfigs_for_sourcing/osourced_not_existing"
    orsource "kconfigs_for_sourcing/orsourced"
    orsource "kconfigs_for_sourcing/orsourced_not_existing"
