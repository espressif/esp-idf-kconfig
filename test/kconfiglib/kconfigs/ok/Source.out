#
# Automatically generated file. DO NOT EDIT.
# Espressif IoT Development Framework (ESP-IDF)  Project Configuration
#
# default:
CONFIG_SOURCED=y

#
# rsourced menu
#
# default:
CONFIG_RSOURCED=y
# end of rsourced menu

# default:
CONFIG_OSOURCED=y
# default:
CONFIG_ORSOURCED=y
